package.json

```diff
@@ -9,6 +9,7 @@
   },
   "main": "index.js",
   "scripts": {
+    "install": "npm run build:dll",
```

Comment:
I can remove it, but I thought it made sense to do whenever we do an npm install since you theoretically run install infrequently and that would be when you upgrade Angular and what not.

It also makes a more conventional build process in my opinion since people only need to run

```
npm install
npm run build
```